src/App.js

```diff
@@ -1,22 +1,57 @@
-import { FormSetDates } from "./components/Form/FormSetDates";
-
+import {FormInput} from './components/activity/FormInput';
+import {ActivityList} from './components/activity/ActivityList';
+import {useState} from 'react';
+import styled from 'styled-components';
+import {ThemeProvider} from 'styled-components';
+import {StyleHeader} from './components/activity/styles/Header.styles';
 
 
+const theme = {
+  colors: {
+    header: 'white',
+    body: 'rose',
+    footer: 'lightgrey',
+  },
+};
 
 export default function App() {
-  return (
-    <>
-    <heading>
-      <h1>Struct your Day</h1>
-    </heading>
-    
-    
-    
+  const [activities, setActivity] = useState([]);
```

Comment:
You should name you set function like your state variable just with set before. If there are activities inside your state, you should name your set function also like this.